scripts/go-tool.sh

```diff
@@ -64,12 +64,10 @@ fi
 function invoke_go() {
   tool="$1"
   shift
-  # TODO(ROX-27054): Remove the redundant strictfipsruntime option if one is found to be so
-  if [[ -z "$GOTAGS" ]]; then
-    GOTAGS="$GOTAGS,strictfipsruntime"
-  else
-    GOTAGS="strictfipsruntime"
-  fi
+
+  # ${parameter:+word}: If parameter is null or unset, nothing is substituted, otherwise the expansion of word is substituted.
+  # - https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion
+  GOTAGS="${GOTAGS:+${GOTAGS},}strictfipsruntime"
```

Comment:
Please retain the todo
```suggestion
  # TODO(ROX-27054): Remove the redundant strictfipsruntime option if one is found to be so
  GOTAGS="${GOTAGS:+${GOTAGS},}strictfipsruntime"
```